migrations/Version20231006120901.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
This seems like it should be removed.
Why was it added?

Reply:
It was added in https://github.com/os2display/display-api-service/pull/165/commits/30ca376725da339545a76db51ef9cf07d9bbcdc5 to make our “Validate Doctrine Schema” stop complaining. It sure is weird …

Reply:
A comment on this weird stuff added in https://github.com/os2display/display-api-service/pull/165/commits/d453c118d03a6b1c8b7b64d1faba7e6b1897d5cf